vpc-native/main.tf

```diff
@@ -1,11 +1,12 @@
 resource "google_container_cluster" "cluster" {
-  name               = var.name
-  location           = var.region
-  min_master_version = var.kubernetes_version
-  network            = var.network_name
-  subnetwork         = var.nodes_subnetwork_name
-  monitoring_service = var.monitoring_service
-  logging_service    = var.logging_service
+  name                     = var.name
+  location                 = var.region
+  min_master_version       = var.kubernetes_version
+  network                  = var.network_name
+  subnetwork               = var.nodes_subnetwork_name
+  monitoring_service       = var.monitoring_service
+  logging_service          = var.logging_service
+  vertical_pod_autoscaling = var.vpa_enabled
```

Comment:
I think this needs to be a block, not just a true/false. 

Documentation here:
https://www.terraform.io/docs/providers/google/r/container_cluster.html

Also, is this still a beta provider feature? If so, we might have to do something to handle that.

Reply:
I don't see that it's beta in the description. Some other parameters specify `(Optional, Beta)`.

Reply:
<img width="513" alt="Screen Shot 2020-10-22 at 1 24 12 PM" src="https://user-images.githubusercontent.com/7624765/96920052-e588c700-1469-11eb-960b-7a95f463113d.png">

Reply:
<img width="819" alt="Screen Shot 2020-10-22 at 1 24 54 PM" src="https://user-images.githubusercontent.com/7624765/96920137-0224ff00-146a-11eb-9ef3-a778b2d90bf4.png">

Reply:
Ah, I see. Missed it. Was looking at this reference:

```The vertical_pod_autoscaling block supports:

    enabled (Required) - Enables vertical pod autoscaling
```